dapp-core/src/main/kotlin/jp/co/soramitsu/dapp/main.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright D3 Ledger, Inc. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+@file:JvmName("DappMain")
+
+package jp.co.soramitsu.dapp
+
+import com.github.kittinunf.result.Result
+import com.github.kittinunf.result.failure
+import mu.KLogging
+import org.springframework.context.annotation.AnnotationConfigApplicationContext
+import org.springframework.context.annotation.ComponentScan
+
+private val logger = KLogging().logger
```

Comment:
companion object : KLogging()

Reply:
`main` function is not related to `DappApplication` so I don't think this is the case